lib/blockscore/actions/update.rb

```diff
@@ -0,0 +1,36 @@
+module BlockScore
+  module Actions
+    module Update
+      def save
+        self.class.patch "#{self.class.endpoint}#{id}", filter_params
+        true
+      rescue
+        false
+      end
+
+      # Filters out the non-updateable params
+      def filter_params
+        persistent = [:id, :object, :created_at, :updated_at, :livemode, :class]
+        @attrs.reject { |k,v| persistent.include?(k) }
```

Comment:
Lint/UnusedBlockArgument: Unused block argument - `v`. If it's necessary, use `_` or `_v` as an argument name to indicate that it won't be used.<br>Style/SpaceAfterComma: Space missing after comma.